docs/versioning.md

```diff
@@ -56,3 +55,15 @@ It MAY include patch level changes. Patch version MUST be reset to 0 when minor
 Major version **X** MUST be incremented for a release that introduces backward incompatible changes.
 A major release can also include minor and patch level changes.
 You must reset the patch and minor version to 0 when you change the major version.
+
+## MFTF version table
```

Comment:
The section title doesn't describe its substance. The topic itself is about MFTF versioning and we see that it contains a table. The heading should give an idea of what is in that table.
Consider renaming to something like:
**Relation to Magento releases**

Reply:
Fixed.